admin/api_programmatic_api_keys.go

```diff
@@ -380,7 +380,7 @@ func (a *ProgrammaticAPIKeysApiService) AddProjectApiKeyWithParams(ctx context.C
 	}
 }
 
-func (r AddProjectApiKeyApiRequest) Execute() (*ApiKeyUserDetails, *http.Response, error) {
+func (r AddProjectApiKeyApiRequest) Execute() (interface{}, *http.Response, error) {
```

Comment:
this change does not seem intentional

Reply:
Never mind, this change is expected: [CLOUDP-260441](https://jira.mongodb.org/browse/CLOUDP-260441)